### PR TITLE
Release doc to use update_docs.sh

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

**Review by elharo — APPROVED** [review]

File: boms/cloud-oss-bom/RELEASING.md [hunk 1]
```diff
@@ -95,13 +95,10 @@ new release is available on Maven Central.
...
 * Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
 * In google3 run:
-    * `$ g4d -f bom`
-    * `$ OLD=19.0.0; NEW=19.1.0`
-      (Adjust the old and new versions)
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version>${OLD}&lt;/version>" "&lt;version>${NEW}&lt;/version>"`
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;${OLD}&lt;/version&gt;" "&lt;version>${NEW}&lt;/version>"`
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;${OLD}&lt;/version>" "&lt;version>${NEW}&lt;/version>"`
-    * Sanity check the cl and send it for review.
+    * `$ scripts/update_doc.sh <old version> <new version>`
```
Comment (elharo) [inline boms/cloud-oss-bom/RELEASING.md:98]: update_docs.sh

**Review by elharo — CHANGES_REQUESTED** [review]

File: boms/cloud-oss-bom/RELEASING.md (hunk 1 above)
Comment (elharo) [reply boms/cloud-oss-bom/RELEASING.md:98]: $ bash ./scripts/update_docs.sh 20.8.0 20.9.0

Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md]: Good catch. Fixed.

**Comment by suztomo** [comment]
@elharo PTAL.

File: boms/cloud-oss-bom/RELEASING.md [hunk 2]
```diff
@@ -95,13 +95,10 @@ new release is available on Maven Central.
...
-    * `$ g4d -f bom`
-    * `$ OLD=19.0.0; NEW=19.1.0`
-      (Adjust the old and new versions)
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version>${OLD}&lt;/version>" "&lt;version>${NEW}&lt;/version>"`
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;${OLD}&lt;/version&gt;" "&lt;version>${NEW}&lt;/version>"`
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;${OLD}&lt;/version>" "&lt;version>${NEW}&lt;/version>"`
-    * Sanity check the cl and send it for review.
+    * `$ scripts/update_docs.sh <old version> <new version>`
+      * For example, `$ scripts/update_docs.sh 16.3.0 16.4.0`
+      * When asked whether to add changes to the first CL, answer yes.
```
Comment (elharo) [inline boms/cloud-oss-bom/RELEASING.md:100]: Do we need this step? I don't think I've been asked this?

Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md:100]: Strange. I was asked by the 2nd invocation of `/google/src/head/depot/google3/devtools/scripts/replace_string`.

Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md:100]: Maybe there were no match for the 2nd and 3rd replace_string command when you ran it.

**Review by elharo — APPROVED** [review]

**Pull request merged**